### PR TITLE
[C# DotNet2] List Subjects

### PR DESCRIPTION
**Pull request by scottf**
https://github.com/ConnectEverything/nats-by-example/issues/217

### CODE REVIEW

File: examples/jetstream/list-subjects/dotnet2/Main.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+await js.PublishAsync("star.2", "star2");
+
+jsStream = await js.GetStreamAsync(stream, new StreamInfoRequest() { SubjectsFilter = ">" });
+Console.WriteLine("Wildcard subjects show the actual subject, not the template:");
+if (jsStream.Info.State.Subjects != null)
+{
+    foreach (var subj in jsStream.Info.State.Subjects.Keys)
+    {
+        Console.WriteLine($"  Subject '{subj}', Count {jsStream.Info.State.Subjects[subj]}");
+    }
```
Comment (mtmk) [inline examples/jetstream/list-subjects/dotnet2/Main.cs:75]:
```suggestion
    foreach (var (subject, count) in jsStream.Info.State.Subjects)
    {
        Console.WriteLine($"  Subject '{subject}', Count {count}");
    }
```

Comment (scottf) [reply examples/jetstream/list-subjects/dotnet2/Main.cs:75]: fixing

**Review by mtmk — CHANGES_REQUESTED** [review]
it'd be nice to use dictionary deconstruction.

**Review by mtmk — APPROVED** [review]
LGTM

**Review by mtmk — APPROVED** [review]

**Review by mtmk — APPROVED** [review]

**Pull request merged**